src/main/java/uk/gov/companieshouse/itemhandler/mapper/OrderDataToCertificateOrderConfirmationMapper.java

```diff
@@ -125,6 +126,37 @@ default String getCertificateRegisteredOfficeOptions(final Item certificate) {
         }
     }
 
+    default String[] getCertificateDirectorOptions(final Item certificate) {
+        final CertificateItemOptions options = (CertificateItemOptions) certificate.getItemOptions();
+        final DirectorOrSecretaryDetails directors = options.getDirectorDetails();
+        final List<String> includes = new ArrayList<>();
+        if (directors == null || directors.getIncludeBasicInformation() == null) {
+            //DO NOTHING
```

Comment:
If this does nothing can we return here and remove the if-else surrounding the method content?